docs/azure_arc_jumpstart/azure_arc_servers/azure_stack_hci/azure_stack_hci_windows/_index.md

```diff
@@ -0,0 +1,226 @@
+---
+type: docs
+title: "Windows Server Virtual Machine"
+linkTitle: "Windows Server Virtual Machine"
+weight: 1
+description: >
+---
+
+## Deploy a Windows Server Virtual Machine in Azure Stack HCI and connect it to Azure Arc using Powershell
+
+The following README will guide you on how to use the provided PowerShell script to deploy a Windows Server Virtual Machine on an [Azure Stack HCI](https://docs.microsoft.com/en-us/azure-stack/hci/overview) cluster and connect it as an Azure Arc enabled server.
+
+This guide will **not** provide instructions on how to deploy and set up Azure Stack HCI and it assumes you already have it provisioned. If you don't have any Azure Stack HCI cluster created, please have a look at the following [Azure Stack HCI 20H2 Evaluation Guide](https://github.com/Azure/AzureStackHCI-EvalGuide).
+
+The commands below and the Powershell script described in this guide should be run on the **management computer**.
```

Comment:
```suggestion
## Deploy a Windows Server Virtual Machine in Azure Stack HCI and connect it to Azure Arc using Powershell

The following README will guide you on how to use the provided PowerShell script to deploy a Windows Server Virtual Machine on an [Azure Stack HCI](https://docs.microsoft.com/en-us/azure-stack/hci/overview) cluster and connect it as an Azure Arc enabled server.

This guide will **not** provide instructions on how to deploy and set up Azure Stack HCI and it assumes you already have it provisioned. If you don't have any Azure Stack HCI cluster created, please have a look at the following [Azure Stack HCI 20H2 Evaluation Guide](https://github.com/Azure/AzureStackHCI-EvalGuide).

This guide will not provide instructions on how to deploy and set up Azure Stack HCI and it assumes you already have a configured cluster. The commands described in this guide should be run on the management computer or in a host server in a cluster.
```

Reply:
@likamrat, thanks for the suggestion. The script was built for running from a management computer only. It's quite difficult to create a script that could be run from the management computer and the host server, because a remote PS session to the host server and its guest VMs is needed